### PR TITLE
TELCODOCS-415: Use of openshift-sdn with sno is deprecated

### PR DESCRIPTION
**Pull request by kdrake20**
Fixes: [TELCODOCS-415](https://issues.redhat.com/browse/TELCODOCS-415)

For: Version 4.11 (see [PR 45840](https://github.com/openshift/openshift-docs/pull/45840) for versions 4.9 & 4.10)

Doc Preview: https://deploy-preview-45830--osdocs.netlify.app/openshift-enterprise/latest/networking/openshift_sdn/about-openshift-sdn.html

Signed-off-by: Katie Tothill ktothill@redhat.com

### CODE REVIEW

**Comment by ktothill** [comment]
**Date and Time:** May 18, 2022  14:12

**Summary:** Created [PR 45830](https://github.com/openshift/openshift-docs/pull/45830).

 
**Next Steps:** Request SME and QE reviews.

**Comment by ktothill** [comment]
@romfreiman, I see that you're the SME for TELCODOCS-415. Could you please provide an SME review for this content change? Thank you!

File: modules/install-sno-about-installing-on-a-single-node.adoc [hunk 1]
```diff
@@ -7,3 +7,8 @@
 = About OpenShift on a single node
 
 You can create a single node cluster with standard installation methods. {product-title} on a single node is a specialized installation that requires the creation of a special ignition configuration ISO. The primary use case is for edge computing workloads, including intermittent connectivity, portable clouds, and 5G radio access networks (RAN) close to a base station. The major tradeoff with an installation on a single node is the lack of high availability.
+
+[IMPORTANT]
+====
+The use of OpenShift software-defined networking (SDN) with Single Node Openshift (SNO) is deprecated. OVN-Kubernetes is the default networking solution for SNO deployments.
```
Comment (aireilly) [inline modules/install-sno-about-installing-on-a-single-node.adoc:13]:
```suggestion
The use of OpenShift software-defined networking (SDN) with single-node Openshift is deprecated. OVN-Kubernetes is the default networking solution for single-node Openshift deployments.
```

File: modules/install-sno-generating-the-discovery-iso-manually.adoc [hunk 2]
```diff
@@ -105,17 +105,19 @@ sshKey: |
 +
 <4> Set the `metadata` name to the cluster name.
 +
-<5> Set the `clusterNetwork` CIDR.
+<5> Set the `networkType`. OVNKubernetes is the default networking for SNO. SDN is deprecated.
```
Comment (aireilly) [inline modules/install-sno-generating-the-discovery-iso-manually.adoc:108]:
```suggestion
<5> Set the `networkType`. OVN-Kubernetes is the default networking provider for single-node clusters. OpenShift SDN is deprecated.
```

File: networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc [hunk 3]
```diff
@@ -12,6 +12,11 @@ OVN-Kubernetes is based on Open Virtual Network (OVN) and provides an overlay-ba
 A cluster that uses the OVN-Kubernetes network provider also runs Open vSwitch (OVS) on each node.
 OVN configures OVS on each node to implement the declared network configuration.
 
+[NOTE]
+====
+OVN is the default networking solution for SNO deployments.
```
Comment (aireilly) [inline networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc:17]:
```suggestion
OVN-Kubernetes is the default networking solution for single-node cluster deployments.
```

**Review by aireilly — COMMENTED** [review]
Added some comments :)

File: modules/install-sno-about-installing-on-a-single-node.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (ktothill) [reply modules/install-sno-about-installing-on-a-single-node.adoc:13]: Updated as suggested.

Comment (ktothill) [reply modules/install-sno-about-installing-on-a-single-node.adoc:13]: Updated as suggested.

File: modules/install-sno-generating-the-discovery-iso-manually.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (ktothill) [reply modules/install-sno-generating-the-discovery-iso-manually.adoc:108]: Updated as suggested.

Comment (ktothill) [reply modules/install-sno-generating-the-discovery-iso-manually.adoc:108]: Updated as suggested.

File: networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc (hunk 3 above)
Comment (ktothill) [reply networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc:17]: Updated as suggested.

**Comment by ktothill** [comment]
**Who:** Tal Hilzenrat, QE
**Contact Method:** Slack, Direct Message
**Date and Time:** May 23, 2022  16:12
**Summary:** Reached out to Tal via Slack, since I couldn't find his username in Github. Requested QE review.

**Next Steps:** Review Tal's changes and implement if needed.

**Comment by ktothill** [comment]
**Contact Method:** Slack, [#forum-docs-review](https://coreos.slack.com/archives/C025XCRRH28/p1653505662242889)
**Date and Time:** May 25, 2022  15:07
**Summary:** Received `lgtm` from:
- SME
- QE

**Next Steps:** Requested review by Peer Review Squad via Slack. Will review their suggestions and implement if needed. Once approved, doc should be ready to be merged.

**Review by sheriff-rh — APPROVED** [review]
LGTM! It looks like netlify didn't get a chance to update the "Generating the discovery ISO manually" section (given it's disabled now), however, those changes make sense to me.

**Pull request merged**